storage/innobase/xtrabackup/doc/source/innobackupex/throttling_ibk.rst

```diff
@@ -1,15 +0,0 @@
-========================================
- Throttling backups with |innobackupex|
-========================================
-
-Although |innobackupex| does not block your database's operation, any backup can add load to the system being backed up. On systems that do not have much spare I/O capacity, it might be helpful to throttle the rate at which |innobackupex| reads and writes |InnoDB| data. You can do this with the :option:`--throttle` option.
```

Comment:
this section is valid for xtrabackup

Reply:
Actually three is a duplicate in the advanced directory. Removed this section from innobackupex